index.js

```diff
@@ -71,8 +72,12 @@ module.exports = (moduleObject, options) => {
 		}
 
 		if (mainProcessPaths.has(path.join(cwd, filePath))) {
-			electron.app.relaunch();
-			electron.app.exit(0);
+			if (!isReloading) {
+				electron.app.relaunch();
+				electron.app.exit(0);
+			}
+
+			isReloading = true;
```

Comment:
```suggestion
			isRelaunching = true;
```

would be more correct